### DISCUSSION TITLE
move mouse with pybricks

### DISCUSSION THREAD

**Discussion by MHking1387** [pybricks/support#1605] [Q&A]
hi . 
i have one technic hub and it has pybricks firmware . 
i want to make something to move mouse with gyro on technic hub 
but i cant use libraries like "pyautogui" in pybricks .
what can i do?
i try to connect hub to python with pybricksdev but i get eror :
no moudle found 'pybricksdev'
but when hub is not connected , other python programs use pyautogui sucsesfully . 
how can i solve that ? 
can we import libraries on pybricksweb on desctop ? or how to use libraries on hub when it connect to python?

**Comment by laurensvalk** [comment]
You could make a project like this: https://pybricks.com/projects/tutorials/wireless/hub-to-device/pc-communication/

Your computer will run real Python, and your hub could print out information like the tilt values, which you could use to move the mouse.
